documentation/modules/security/proc-accessing-kafka-using-loadbalancers.adoc

```diff
@@ -100,16 +100,16 @@ status:
     - addresses:
         - host: >-
             a8d4a6fb363bf447fb6e475fc3040176-36312313.us-west-2.elb.amazonaws.com
-          port: 9095
+          port: 9094
       bootstrapServers: >-
-        a8d4a6fb363bf447fb6e475fc3040176-36312313.us-west-2.elb.amazonaws.com:9095
+        a8d4a6fb363bf447fb6e475fc3040176-36312313.us-west-2.elb.amazonaws.com:9094
       certificates:
         - |
           -----BEGIN CERTIFICATE-----
           
           -----END CERTIFICATE-----
-      name: external
-      type: external
+      name: external3
+      type: external3
```

Comment:
```suggestion
      type: loadbalancer
```

Reply:
Same as above, this is wrong. It should be `external3`.